packages/plans-grid/src/plans-feature-list/style.scss

```diff
@@ -105,6 +106,19 @@ ul.plans-feature-list__item-group {
 			}
 		}
 	}
+
+	// Disabled/unavailable text color for when item is not a button
+	// (but is a disabled feature)
+	.plans-feature-list__item--disabled-message & {
```

Comment:
Can't we move the text styling a level deeper (i.e. `.plans-feature-list__item plans-feature-list__item--disabled-message > .plans-feature-list__item-content-wrapper`) to be more explicit about which elements are addressed and to ease our specificity struggles?

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/49523/commits/a519350d2d75284544f38e5887f495dd2617b87c